vcluster/deploy/topologies/air-gapped.mdx

```diff
@@ -78,6 +78,116 @@ The `vCluster` Helm chart is a tarball that contains all the necessary images
 to deploy the vCluster control plane.
 :::
 
+### Deploy vCluster using a private web server
```

Comment:
This whole section should be merged with the platform air-gapped doc.

Reply:
What does merging mean?